src/screens/HyperSwitch/Disputes/ShowDisputes.res

```diff
@@ -33,63 +33,184 @@ module DisputesNoteComponent = {
+        popUpType: (Warning, WithIcon),
+        heading: "Accept this dispute?",
+        description: "By accepting you will lose this dispute and will have to refund the amount to the user. You won’t be able to submit evidence once you accept"->React.string,
+        handleConfirm: {text: "Proceed", onClick: _ => handleAcceptDispute()->ignore},
+        handleCancel: {text: "Cancel"},
+      })
+    }
+
+    <OrderUtils.Section
+      customCssClass={`border border-jp-gray-940 border-opacity-75 dark:border-jp-gray-960 ${bgColor} rounded-md p-6 flex flex-col gap-6`}>
+      <div className="flex items-center justify-between">
+        <div className="flex gap-2 items-center">
+          <p className="font-bold text-3xl">
+            {DisputesEntity.amountValue(
+              data.amount,
+              data.currency->String.toUpperCase,
+            )->React.string}
+          </p>
           {getStatus(data)}
         </div>
-        <FormRenderer.DesktopRow>
-          <div
-            className={`flex flex-wrap ${justifyClassName} dark:bg-jp-gray-lightgray_background dark:border-jp-gray-no_data_border`}>
-            {detailsFields
-            ->Array.mapWithIndex((colType, i) => {
-              <UIUtils.RenderIf condition={!(excludeColKeys->Array.includes(colType))}>
-                <div className={`flex ${widthClass} items-center`} key={Belt.Int.toString(i)}>
-                  <OrderUtils.DisplayKeyValueParams
-                    heading={getHeading(colType)}
-                    value={getCell(data, colType)}
-                    customMoneyStyle="!font-normal !text-sm"
-                    labelMargin="!py-0 mt-2"
-                    overiddingHeadingStyles="text-black text-sm font-medium"
-                    textColor="!font-normal !text-jp-gray-700"
-                  />
-                </div>
-              </UIUtils.RenderIf>
-            })
-            ->React.array}
+        <UIUtils.RenderIf
+          condition={disputeEvidenceUpload && data.dispute_status === "dispute_opened"}>
+          <div className="flex gap-4">
+            <Button
+              buttonType={Secondary}
+              text="Accept Dispute"
+              buttonSize={Small}
+              customButtonStyle="!py-3 !px-2.5"
+              onClick={_ => handlePopupOpen()}
+            />
+            <Button
+              buttonType={Primary}
+              text="Counter Dispute"
+              buttonSize={Small}
+              customButtonStyle="!py-3 !px-2.5"
+              buttonState={Disabled}
+            />
           </div>
-        </FormRenderer.DesktopRow>
-        <UIUtils.RenderIf condition={children->Option.isSome}>
-          {children->Option.getWithDefault(React.null)}
         </UIUtils.RenderIf>
-      </OrderUtils.Section>
-    }
+      </div>
+      <div className="h-px w-full bg-grey-200 opacity-30" />
+      <UIUtils.RenderIf
+        condition={disputeEvidenceUpload && data.dispute_status === "dispute_opened"}>
```

Comment:
Ask backend for types so we can create a mapper

Reply:
Addressed